tutorials/cloud-functions-sns-pubsub/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Connect Google Cloud Pubsub to AWS SNS topics through Cloud Functions
+description: Learn how to integrate Google Cloud Pubsub with AWS SNS using
+Google Cloud Functions.
+author: ptone
+tags: Cloud Functions, AWS, SNS, Node.js
+date_published: x
+---
+## Introduction
+
+This tutorial and sample demonstrates using [Google Cloud Functions][functions]
+to act as an integration point between the AWS Simple Notification Service
+(SNS) and Google Cloud Pubsub.  The Cloud Function is implemented in
```

Comment:
In this case, since other languages may be available, need to discriminate from "the function" and "Cloud Functions" the product.  I'll change to "the function" to avoid confusion